testpilot/settings.py

```diff
@@ -509,7 +509,8 @@ def lazy_langs():
     "https://code.cdn.mozilla.net/fonts/fira.css":
         "sha384-APhs/OUouhH+ZbBANL3+7a5J1sVSYyfUhGxTWiDMkTuEIO5fXWZonzNEj+CagDB7",
 }
-STATICINTEGRITY_CACHE_TTL = 30
+STATICINTEGRITY_CACHE_TTL = config('STATICINTEGRITY_CACHE_TTL',
```

Comment:
Yeah, just making the value configurable, so I can set it to a very short duration on dev where the static files change very often during development